src/utils/gitConstants.js

```diff
@@ -3,5 +3,11 @@ module.exports.ADDITION = 'A'
 module.exports.DELETION = 'D'
 module.exports.GIT_DIFF_TYPE_REGEX = /^.\s+/u
 module.exports.MINUS = '-'
+module.exports.PERMISSIVE_DIFF_PARAMS = [
+  '--ignore-all-space',
+  '--ignore-blank-lines',
+  '--ignore-cr-at-eol',
+  '--word-diff-regex',
```

Comment:
@nvuillam According to the [git documentation](https://git-scm.com/docs/diff-options#Documentation/diff-options.txt---word-diff-regexltregexgt), the `--word-diff-regex` parameter expects a `<regex>` value.
Otherwise it may apply a regex that is set locally in your gitattributes or git-config (extract from doc: "_The regex can also be set via a diff driver or configuration option, see gitattributes[5] or git-config[1]. Giving it explicitly overrides any diff driver or configuration setting._").

To make sure the behaviour of this command is consistent for all SGD users, what do you think about specifying explicitly the regex to use? In other words, I suggesting to modify line 10 to specify explicitly the `<regex>` in `--word-diff-regex=<regex>` (I guess you could look into you gitattributes and git-config to retrieve the value that you used during your tests).

@scolladon To go further, in addition to specifying explicitly the default regex to apply, we could even offer the ability for the users to override it with their own regex (but that would a "nice-to-have" feature, while ensure a consistent behaviour in the command in more important IMHO).

Reply:
@mehdisfdc I love the id to let the user override the default regex, I will implement it with the new parameter name (once everyone is ok) and with the default regex value

Reply:
More flexibility for the user, I fully agree, great suggestion :)

Reply:
I bounce on @mehdisfdc's message, @nvuillam could you give us the git config taken by `--word-diff-regex`?

Reply:
I'm sorry, I just used the parameter and I think I viewed a difference compared to without, but i've not sent any value to this parameter

Reply:
Could you retry on your side to make sure it does something ?
And if it does something on your side, as you don't use any regex with it, could you send us your gitattributes and git config please ?

Reply:
I don't have any .gitattributes defined

About git config, except user.name & user.email, I use 
- core.longpaths=true
- core.quotepath=false

I'll try to make new tests, but unfortunately my work week is already busy, it may wait until next week :/

Reply:
But why not publishing as it is now, and send --word-diff-regex to git if --word-diff-regex is sent to sfdx-git-delta, and append --word-diff-regex value only if it is sent ? :)

Reply:
I prefer not to do that because if no parameter are given to `--word-diff-regex` then it uses the local configuration in `.gitattributes` and `.git/config`, which can be anything and could lead to unexpected behaviour

Reply:
So, I poced around and play a little bit with it.
It is also true for me that `--word-diff-regex` is required to have the right behavior.
I dig a little bit more in the documentation of this parameter in git and I found that there are [default regex](https://stackoverflow.com/questions/30428377/what-are-git-diff-word-diff-default-regexps) used (which are happened to the override foundable in `.gitattributes` and `.git/config`)

I wonder if we will have issues about this and how it will be to debug 😅 but I don't think it will be used by most of the users.
With that knowledge, it looks good to me 😄